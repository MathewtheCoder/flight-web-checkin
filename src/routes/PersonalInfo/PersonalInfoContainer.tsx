import React from 'react'
import PersonalInfoForm from './components/PersonalInfoForm'
import {useHistory} from 'react-router-dom'

type PersonalInfoState = {
    last_name: string;
    flight_no: string;
};
const PersonalInfoContainer = () => {
    const history = useHistory<PersonalInfoState>()
    return (
        <main className="webcheckin">
           <div className="formWrapper">
            <h1>Hi, {history?.location?.state?.last_name}!</h1>
            <PersonalInfoForm />
           </div>
       </main>
    )
}

export default PersonalInfoContainer