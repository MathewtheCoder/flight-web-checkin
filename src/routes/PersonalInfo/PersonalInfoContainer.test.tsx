import React from 'react';
import { shallow, toJson } from 'tests/enzyme';
import PersonalInfoContainer from './PersonalInfoContainer';

describe('PersonalInfoContainer tests', () => {

  it('renders correctly PersonalInfoContainer page', () => {
    const wrapper = shallow(<PersonalInfoContainer />);
    // Expect the wrapper object to be defined
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});