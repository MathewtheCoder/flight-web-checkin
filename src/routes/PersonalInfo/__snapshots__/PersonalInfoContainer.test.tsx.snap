// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalInfoContainer tests renders correctly PersonalInfoContainer page 1`] = `
<main
  className="webcheckin"
>
  <div
    className="formWrapper"
  >
    <h1>
      Hi, 
      !
    </h1>
    <PersonalInfoForm />
  </div>
</main>
`;
